### ISSUE TITLE
3.1 Animated Headline Bug

### ISSUE THREAD

**Issue by RedboatDesign** [elementor/elementor#13951]
After the release of 3.1. The New Animated Headline feature breaks the rotating text feature.

**How to replicate.**

1. Create an animated headline with Rotating, Typing and enter several Rotating text items.
2. Save and go view page and then scroll down a little and then back up
3. You will see that the rotating text starts appearing / doubling up on one line.

Elementor 3.09 does not have the issue for those who want to roll back.

This has been reported to support and people within the FB community have seen the same issue.

**Comment by jormeer** [comment]
I also had that. I fixed it by clearing my cache. (Also browser cache) And browser cache when using elementor editor.

**Comment by RedboatDesign** [comment]
I tried a full cache clearance but will try again. Not sure I ran a browser cache clearance.

**Comment by RedboatDesign** [comment]
@jormeer Just updated and clears ALL caches, Site, Browser and Server. 

Still getting the issue. See screen grab. \
 WEBSITE and CREATIVE are on Separate lines in the editor. But join in 3.1.

![Screen Shot 2021-02-18 at 12 02 35](https://user-images.githubusercontent.com/53269948/108354365-412d4c00-71e1-11eb-9359-27db8e08f7bf.png)

**Comment by RedboatDesign** [comment]
Reverted back to 3.09 and all good again.

**Comment by RedboatDesign** [comment]
Another note to add. I have also updated my development site to 3.1 and get the same issue. 

One other note is that I do not have BEFORE or AFTER tex.

It is important that you need to scroll down and up again and wait. On page load all is fine, it is only after scrolling the site you will find the typing text will start showing the bug.

**Comment by Grassmedia** [comment]
Definitely not cache related.  On page load the animated headline displays correctly.  However, scroll down the page and back up and the animated portion of the text displays twice.  Confirmed on two sites.

**Comment by mister-mst** [comment]
I have the same error.

**Comment by JustinM2M** [comment]
Same here

**Comment by RedboatDesign** [comment]
**REPONSE FROM ELEMENTOR**

Thanks for contacting us and hope you are doing well and staying safe.

I would like to apologize for the inconvenience caused and to let you know that we've received similar reports about this issue recently. Our developers are already aware of this issue caused by the recent Elementor Pro 3.1.0 update as you already observed, and it should be patched in one of the upcoming updates.

Meanwhile, you can keep using Elementor Pro 3.0.9 or better 3.0.10 where it will work as expected. Rolling Back To a Previous Version of Elementor

We again apologize for the inconvenience!

**Comment by perezmunoz** [comment]
Same issue here. I have 3 rotating text with a Slide style and 2 seconds duration.

**Comment by shilo-ey** [comment]
Hey all,

This issue has been resolved in Elementor Pro v3.2.0

Feel free to update!

**Comment by webyeshua** [comment]
I don't have a problem with Rotating headline, but I have similar problem with Highlighting headline. The highlighted headline text repeated itself, this is true in both Elementor Pro v3.1.0 and v.3.2.0.

**Comment by webyeshua** [comment]
This bug is not yet fixed in 3.2.1. It is tested that the problem still happen to Safari on iOS, Chrome on Android, Miscrosoft Edge or Chrome on Windows 10, Firefox on Mac. But it is working fine to Safari, Chrome and Opera on my Mac. Of course, cache are clear before testing. Still waiting for the fix...

**Comment by faisalbinali12** [comment]
This Bug is not fixed yet in 3.2.1. I tried many many ways. After scrolling it maseup with other lines. That's why I give this widget max height and overflow hidden. After that, It won't fix, but it's won't overlap.

**Comment by Chaoscontrol** [comment]
Can we reopen this? It is still happening on 3.2.2. 

My case is Rotating, slide down style. I created a test page to showcase it here: https://dev.nereapr.com/test/

**Comment by FunkyCss** [comment]
This bug is still in the latest version of Elementor

**Comment by Elthibert** [comment]
same problem...

**Comment by CentralCrypto** [comment]
frontend is showing break lines here !!
![image](https://github.com/elementor/elementor/assets/132160255/d23df565-d510-4c00-9e53-8c4df7a1d830)
